packages/aws-rfdk/lib/lambdas/nodejs/configure-spot-event-plugin/test/handler.test.ts

```diff
@@ -445,6 +465,10 @@ describe('SEPConfiguratorResource', () => {
 
       expect(mockConfigureSpotEventPlugin.mock.calls.length).toBe(1);
       expect(mockConfigureSpotEventPlugin.mock.calls[0][0]).toEqual([...configs, ...securitySettings]);
+
+      expect(mockAddGroups.mock.calls.length).toBe(1);
+
+      expect(mockAddPools.mock.calls.length).toBe(1);
```

Comment:
we should probably assert what arguments are being passed to these functions. How about:

```ts
expect(mockAddGroups).toHaveBeenCalledWith(deadlineGroups);
expect(mockAddPools).toHaveBeenCalledWith(deadlinePools);
```

Reply:
Looks good now 👍